main/getting-started/README.md

```diff
@@ -63,9 +63,15 @@ How to install software that the Agoric SDK depends on.
   documentation, accessible from the Zoe documentation sidebar menu.
 
 When needed, refer to the **[ERTP API Reference](/ertp/api/)**, **[Zoe API
-Reference](/zoe/api/)**, **[Wallet UI Guide and API Reference](/wallet-api.md)**,
-and **[Agoric CLI Guide](/getting-started/agoric-cli-guide.md)** for details about 
-their respective commands. 
+Reference](/zoe/api/)**, **[Wallet Guide and API Reference](/guides/wallet)**,
+and **[Agoric CLI Guide](/guides/agoric-cli/)** for details about 
+their respective commands. The Wallet Guide also covers the Wallet UI and 
+general working with the Agoric Wallet.
+
+You can access a REPL (Read-Eval-Print Loop). From the REPL and its `home` object,
+developers can use all the on-chain commands that deployment scripts use to deploy
+contracts and Dapps. In addition to the on-chain commands, you can also run any 
+other JavaScript commands. See the **[REPL Guide](/repl/)** for details.
```

Comment:
> In addition to the on-chain commands, you can also run any 
other JavaScript commands.

This sounds very strange to me. It's all JavaScript, so speaking of JavaScript as if it is something different is very jarring to the user. I think we can drop the sentence entirely.